ansible/roles/machine/provision/templates/ipa-test-config.yaml

```diff
@@ -26,6 +26,20 @@ domains:
 {% endfor %}
   name: ipa.test
   type: IPA
+
```

Comment:
We have to add this otherwise ipa-run-tests would skip the tests saying it could not find any config for AD tests